extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -711,7 +713,7 @@ AMP.extension(TAG, '0.1', AMP => {
   AMP.registerServiceForDoc(
       'amp-analytics-instrumentation', InstrumentationService);
   AMP.registerServiceForDoc('activity', Activity);
-  installVariableService(AMP.win);
+  installVariableServiceForDoc(AMP.ampdoc);
```

Comment:
`AMP.ampdoc` does not identify the AMP document in a PWA

Reply:
Thanks for reporting. Will work on the fix tomorrow.